packages/pharos-site/src/components/statics/BestPractices.module.css

```diff
@@ -10,22 +10,38 @@
   display: grid;
   grid-template-columns: max-content 1fr;
   column-gap: var(--pharos-spacing-1-x);
+  margin-bottom: var(--pharos-spacing-one-half-x);
 }
 
 .line_dont {
   border: none;
   height: 2px;
   background-color: var(--pharos-color-feedback-error);
-  margin-bottom: var(--pharos-spacing-three-quarters-x);
+  margin-bottom: var(--pharos-spacing-one-half-x);
 }
 
 .line_do {
   border: none;
   height: 2px;
   background-color: var(--pharos-color-feedback-success);
-  margin-bottom: var(--pharos-spacing-three-quarters-x);
+  margin-bottom: var(--pharos-spacing-one-half-x);
+}
+
+.text_do {
+  color: var(--pharos-color-feedback-success);
+  font-weight: bold;
+}
+
+.text_dont {
+  color: var(--pharos-color-feedback-error);
+  font-weight: bold;
 }
 
 .text__guideline {
   color: var(--pharos-color-marble-gray-20);
 }
+
+div div .text__guideline ul > ::marker {
+  color: var(--pharos-color-marble-gray-20);
+  font-size: 10px;
+}
```

Comment:
We generally avoid using element-specific, nested, and direct descendent selectors as much as possible—I recommend adding a new class to the `ul` elements so you can target `.some-new-class ::marker` here instead.

Reply:
While it still changes the sizing. The colour keeps being overwritten by the main selector and I'm not sure how to resolve that or raise the specificity of the current selector.